src/ports/owner/utils.ts

```diff
@@ -24,20 +20,25 @@ export function getOwnersQuery(
   isCount: boolean
 ) {
   const first = isCount ? 1000 : filters.first
```

Comment:
can we modify this as well? 🙏 
```suggestion
  const first = isCount ? MAX_RESULTS : filters.first
```